### DISCUSSION TITLE
More detail on SNMP failures?

### DISCUSSION THREAD

**Discussion by toomuchio** [akvorado/akvorado#755] [Q&A]
Hi,

On the metrics page we can see the counters per exporter for missing interfaces and ifalias/ifdescr/ifspeed is missing, I'm wondering is there a way we can expose more information such as the ifindex that's it's attempting to poll and failing?

Logs again only show that some objects failed to grab but not which, in large deployments it's quite difficult to track down rouge interfaces without attributes assigned.

**Comment by vincentbernat** [comment]
As a workaround, you can use tcpdump to get such an information.

**Comment by toomuchio** [comment]
Hi Vincent,

Thanks for that insight, I'll see what I can do. Is expanding the logging to be more verbose a feature you have planned?

I wouldn't mind sending a PR if you give me some pointers on where to implement / aren't working on it already.

**Reply by vincentbernat** [reply]
Missing OID is now something expected on some setups. See https://github.com/akvorado/akvorado/issues/488. We cannot flood the log with this. You could classify the interface with some special description (including the index) and detect the problem this way.
